mods_cocina_mappings/mods_to_cocina_abstract.txt

```diff
@@ -0,0 +1,70 @@
+1. Single abstract
+<abstract>This is an abstract.</abstract>
+{
+  "note": [
+    {
+      "value": "This is an abstract.",
+      "type": "summary"
+    }
+  ]
+}
+
+2. Multilingual abstract
+<abstract lang="eng" script="Latn" altRepGroup="1">This is an abstract</abstract>
+<abstract lang="rus" script="Cyrl" altRepGroup="1">Это аннотация.</abstract>
+{
+  "note": [
+    {
+      "parallelValue": [
+        {
+          "value": "This is an abstract.",
+          "type": "summary",
+          "language": [

```

Comment:
Why is language an array?  Is that to support a note containing more than one language?

Reply:
It's partly because I tried not to create a situation where the same attribute had different value types depending on what level of the schema it appears at. So I was trying to avoid having language be an array at the top level, but an object at lower levels within the schema. But yes, it is also possible for a value to contain multiple languages.